addons/rearm/functions/fnc_addVehicleMagazinesToSupply.sqf

```diff
@@ -29,16 +29,17 @@ if (_vehicle isEqualType objNull) then {_vehicle = typeOf _vehicle};
 if (_vehicle == "") exitWith {
     ERROR_1("VehicleType [%1] is empty in ace_rearm_fnc_addVehicleMagazinesToSupply",_string);
 };
+private _turrets = [_vehicle] call FUNC(getAllRearmTurrets);
```

Comment:
I still have a few arguments for function left in my pocket.
- Cleaner code. It hides implementation details. `[_veh] call FUNC(getAllRearmTurrets)` tells the reader immediately what the code does, whereas `[[-1]] + allTurrets veh` leaves him to wonder "What is that number -1 doing there?" and it requires a comment to explain what is happening.
- Replacement for `allTurrets`. There really should be a command for retrieving all turrets including the driver turret. I think most people would expect such a command to exist, because it is so common use case in Arma. It however doesn't, so this function is provided as a substitute for how the Arma API should have been.

Now you could argue that a macro would satisfy those needs as well, but I don't really like macros because they work with different rules than the rest of the code.